randovania/games/dread/json_data/Burenia.txt

```diff
@@ -1110,8 +1110,36 @@ Extra - polygon: [[-900.0, 3700.0], [-5000.0, 3700.0], [-5000.0, -2115.0], [-900
       Trivial
   > Pickup (Energy Tank)
       All of the following:
-          Flash Shift and Morph Ball
-          Grapple Beam or Spider Magnet or Walljump (Beginner)
+          Morph Ball
+          Any of the following:
+              All of the following:
+                  Use Spin Boost
+                  Movement (Intermediate) or Slide Jump (Beginner)
+              All of the following:
+                  Speed Booster and Disabled Door Lock Randomizer
+                  Spider Magnet or Movement (Beginner)
+              All of the following:
+                  Flash Shift
+                  Any of the following:
+                      Spider Magnet
+                      All of the following:
+                          Movement (Beginner)
+                          Walljump (Beginner) or Use Spin Boost
+              All of the following:
+                  Grapple Beam
+                  Any of the following:
+                      Spider Magnet
+                      All of the following:
+                          Movement (Beginner)
+                          Walljump (Beginner) or Use Spin Boost
```

Comment:
Grapple can definitely just be a trivial option. It's definitely not hard at all to just stand in place and grapple the top of the spider wall, and wall jump (simple wall jumps like this are being treated as trivial)

Reply:
Nothing is hard when you know how to do it.  You need to stand close enough to the edge, aim high enough at the wall , let go of the grapple at the correct time and then wall jump during a small window. All in all, I think this is a complicated operation that I don't think we can expect less experienced dread players to understand that they can do.